src/main/java/util/ContentType.java

```diff
@@ -0,0 +1,36 @@
+package util;
+
+public enum ContentType {
+    HTML("text/html", "src/main/resources/templates"),
```

Comment:
```suggestion
    HTML("text/html", BASE_PATH + "templates"),
```
BASE_PATH를 상수화하면 어때요?